VMAccess/vmaccess.py

```diff
@@ -244,7 +266,7 @@ def _set_user_account_pub_key(protect_settings, hutil):
 
     # user name must be provided if set ssh key or password
     if not protect_settings or 'username' not in protect_settings:
-        return
+        return ("None", None)
```

Comment:
Are the error codes pre-defined already? If we can, maybe we should add an error code for username error, or bundle this under USER_ERROR.